### ISSUE TITLE
Accessibility Issues in the registration , profile Edit and Login form 

### ISSUE THREAD

**Issue by WOLFIEEEE** [wp-insider/simple-membership#182]
Hey There as discussed , I am accessibility developer. I was recently using your plugin to build a site for a client as I saw those issues . In USA , Europe and Canada Location accessibility is something than many of business are looking into it . If you guys are okay with it . I will create a pull request with fixes as I get some time . 

Is it something Okay with you team ?

Issues can be :
1. Error Message not announced as soon as they appears on screen should be implemented using aria-live or role=alert.
2. unnecessary list implantation for error message : not needed.
3. Error are not linked with the input field properly .

**Comment by wp-insider** [comment]
Sure, you can work on these accessibility-related items. It's best to tackle one item at a time to keep the changes manageable for review and testing. In summary, make small, incremental changes and submit a pull request for each, so we can review them easily and provide feedback if needed.
